src/fsharp/IlxGen.fs

```diff
@@ -5089,7 +5089,7 @@ and GenDecisionTreeSwitch cenv cgbuf inplabOpt stackAtTargets eenv e cases defau
             | _ -> error(InternalError("these matches should never be needed", switchm))
 
 and GenDecisionTreeCases cenv cgbuf stackAtTargets eenv defaultTargetOpt targets repeatSP targetInfos sequel caseLabels cases (contf: Zmap<_,_> -> FakeUnit) =
-    assert(cgbuf.GetCurrentStack() = stackAtTargets) // cgbuf stack should be unchanged over tests. [bug://1750].
+    //assert(cgbuf.GetCurrentStack() = stackAtTargets) // cgbuf stack should be unchanged over tests. [bug://1750].
```

Comment:
Why is this commented out?

Reply:
The assert is triggering in debug builds and is just not accurate

Reply:
Confirming I've hit this assert a bunch of times recently.